config/schedule.rb

```diff
@@ -50,6 +50,13 @@
   rake "reports:export:boxi"
 end
 
+# This is the daily AD renewal letters export service.
+# Will run once a day in the early morning hours and generate a zip file containing
+# data required for boxi.
```

Comment:
```suggestion
# a PDF with the day's assisted digital renewal letters.
```

Reply:
The zip file bit may be inaccurate as well – not sure

Reply:
Copy paste fail! D: